numpy/core/src/npymath/npy_math_private.h

```diff
@@ -477,7 +476,7 @@ do {                                                            \
      ((x).a[LDBL_MANH_INDEX] & ~LDBL_MANH_MASK) |                       \
      (((IEEEl2bitsrep_part)(v) << LDBL_MANH_SHIFT) & LDBL_MANH_MASK))
 
-#endif /* #ifndef HAVE_LDOUBLE_DOUBLE_DOUBLE_BE */
+#endif /* #if !HAVE_LDOUBLE_DOUBLE_DOUBLE_BE and !LE */
```

Comment:
nit: something still not right with this comment, perhaps remove it since the logic is not simple

Reply:
What about 
```C
#endif /* !HAVE_LDOUBLE_DOUBLE_DOUBLE_* */
```

I don't think the goal of the comment isn't to get absolutely the right logic, it's just to mark which `ifdef` the `endif` corresponds to, as done elsewhere in this and other files. 

The C preprocessor `ifdefs` don't nest, so they're hard to read without this kind of comment.

Reply:
`#endif /* !HAVE_LDOUBLE_DOUBLE_DOUBLE_* */` looks good